presto-native-execution/presto_cpp/main/common/Configs.h

```diff
@@ -335,9 +335,13 @@ class SystemConfig : public ConfigBase {
       "cache.velox.ttl-check-interval"};
   static constexpr std::string_view kUseMmapAllocator{"use-mmap-allocator"};
 
+  /// TODO: Deprecate. Please use metrics-reporting-enabled instead.
   static constexpr std::string_view kEnableRuntimeMetricsCollection{
       "runtime-metrics-collection-enabled"};
 
+  static constexpr std::string_view kEnableMetricsReporting{
+      "metrics-reporting-enabled"};
```

Comment:
I wonder if we should name the new config prop something like stats-counters-reporting-enabled?

Otherwise it might still confuse people if we are talking about runtime metrics we report for the individual tasks.